text/0000-customisability.md

```diff
@@ -0,0 +1,86 @@
+- Start Date: 2016-10-25
+- RFC Issue: #3
+- RFC PR: 
+- Implementation Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Customisation of Rustfmt should be allowed (via a `rustfmt.toml` file), but not
+encouraged.
+
+
+# Details
+[details]: #details
+
+A formatter such as Rustfmt may be customised by the user. These customisations
+may be saved for a project using a cusomtisation file. For example, Rustfmt can
+be customised by either a `rustfmt.toml` or `.rustfmt.toml` in their project
+directory or any parent of that directory. Rustfmt will read options from the
+first toml file (scanning up the directory tree). If an option is not present in
+the file, Rustfmt uses the default option (as specified by the code style
+process RFCs). The set of options available is not specified at this time; it
+will be part of the style RFC process to enumerate them.
+
+A formatting tool may be customised in other ways, but must stick to the options
+and defaults specified by the style RFCs.
+
+Customisation will be documented, but explicitly discouraged.
+
+All official Rust projects which use Rustfmt *must* use the default style and
+*must not* include a customisation file.
+
+# Implementation
+[implementation]: #implementation
+
+Rustfmt already allows customisation via `rustfmt.toml`. There is nothing more
+to implement here. Over time, I expect the number of options to be reduced.
```

Comment:
I meant that the RFC process should lead to a reduction, I agree that as time goes by there is likely to be a natural increase. I'll clarify.